proto/ibc/applications/interchain_accounts/v1/query.proto

```diff
@@ -18,9 +18,8 @@ message QueryInterchainAccountAddressRequest {
   option (gogoproto.equal)           = false;
   option (gogoproto.goproto_getters) = false;
 
-  // Owner address is the owner of the interchain account on the controller chain
-  string owner_address = 1;
-  string connection_id = 2;
+  // Counterparty PortID is the portID on the controller chain
+  string counterparty_port_id = 1;
```

Comment:
This seemed like a better request argument, rather than specifying each individual piece and rebuilding the portID

Reply:
My reasoning fore having the previous arguements was that the connection-id represents the chain essentially, so that will be easy to query for. Likewise, it will be easy for whoever is querying to find the owner's address. 

AFAIK we are not able to query for portIDs right now, so it's maybe less ideal? 

I guess, if I'm reasoning this from first principles, who/what is going to be the client that might be using this query? If it is a human using the CLI, maybe it's less likely they'll have the portID on hand. If it's a client-side application using cosmJS or something where would the client get the portID from?

Reply:
Also, just thinking about it, we should probably be able to query this on both the controller and host chain. 

Sidenote: Do we even want/need this query at all?

Reply:
PortID's will always be known after ChanOpenInit is called. This query is stateless so it isn't necessary. We could just have it be a cli command

Reply:
Let's go with this, no point in blocking work on this for the time being.